### DISCUSSION TITLE
How to I convert a Visual Block selection into multiple cursors? (One per line)

### DISCUSSION THREAD

**Discussion by Hubro** [mg979/vim-visual-multi#195] [Q&A] [answered]
If I have a Visual Block selection like this:

![image](https://user-images.githubusercontent.com/597206/140754829-e0f7b010-27a1-4490-a2be-b5427e006e4b.png)

Is there a quick way to convert this selection into one cursor on each line?

(I know I can ctrl+up/down but that's much less practical)

**Comment by mg979** [comment]
:h vm-mappings-visual

`<Plug>(VM-Visual-Cursors)` and `<Plug>(VM-Visual-Add)` should be the plugs (depending on what you want to do). With `Visual Cursors` you don't need a block selection (works from any kind of visual mode).

**Reply by Hubro** [reply]
I've tried both of these by using this keybinding:

```vim
vnoremap <F2> <Plug>(VM-Visual-Cursors)
```

```vim
vnoremap <F2> <Plug>(VM-Visual-Add)
```

But neither of them appear to do anything :thinking: What are they supposed to do?

**Comment by mg979** [comment, marked as answer]
`xmap` not `xnoremap`

    xmap <F2> <Plug>(VM-Visual-Cursors)

**Reply by Hubro** [reply]
Oh crap! Obviously I can't use "nore" mappings when I'm trying to map to a plugin keybind :man_facepalming: 

`<Plug>(VM-Visual-Cursors)` is indeed exactly what I'm looking for. Thanks a lot!

**Reply by huyz** [reply]
Hmm, I did it with `xnoremap` and it works fine for me.
